.npmignore

```diff
@@ -14,3 +14,6 @@ coverage
 resources
 src
 packages
+packages/graphiql/*.html
+cypress.json
+babel.config.js
```

Comment:
Is npmignore required? We should be using the `"files"` entry in `package.json` files which should mean we can completely delete `.npmignore`.

Reply:
ah good point, i fix this partially in the next PR, we can get rid of it totally in another PR later